Tests.Integration.Presentation.Web/Contract/V2/ItContractsApiV2Test.cs

```diff
@@ -367,6 +367,136 @@ public async Task Cannot_POST_With_Duplicate_Name_In_Same_Org()
+        }
+
+        [Fact]
+        public async Task Cannot_POST_With_Parent_From_Other_Organization()
+        {
+            //Arrange
+            var (token, user, organization1) = await CreatePrerequisitesAsync();
+            var organization2 = await CreateOrganizationAsync();
+            var parent = await ItContractHelper.CreateContract(CreateName(), organization2.Id);
+
+            var requestDto = new CreateNewContractRequestDTO()
+            {
+                OrganizationUuid = organization1.Uuid,
+                Name = CreateName(),
+                ParentContractUuid = parent.Uuid
+            };
+
+            //Act
+            using var createResponse = await ItContractV2Helper.SendPostContractAsync(token, requestDto);
+
+            //Assert
+            Assert.Equal(HttpStatusCode.Forbidden, createResponse.StatusCode);
+        }
+
+        [Fact]
+        public async Task Cannot_POST_With_Unknown_Parent()
+        {
+            //Arrange
+            var (token, user, organization1) = await CreatePrerequisitesAsync();
+
+            var requestDto = new CreateNewContractRequestDTO()
+            {
+                OrganizationUuid = organization1.Uuid,
+                Name = CreateName(),
+                ParentContractUuid = A<Guid>()
+            };
+
+            //Act
+            using var createResponse = await ItContractV2Helper.SendPostContractAsync(token, requestDto);
+
+            //Assert
+            Assert.Equal(HttpStatusCode.NotFound, createResponse.StatusCode);
+        }
+
+        [Fact]
+        public async Task Can_PUT_With_Parent()
+        {
+            //Arrange
+            var (token, user, organization) = await CreatePrerequisitesAsync();
+
+            var parentRequestDto = new CreateNewContractRequestDTO()
+            {
+                OrganizationUuid = organization.Uuid,
+                Name = CreateName()
+            };
+
+            var parent = await ItContractV2Helper.PostContractAsync(token, parentRequestDto);
+
+            var requestDto = new CreateNewContractRequestDTO()
+            {
+                OrganizationUuid = organization.Uuid,
+                Name = CreateName()
+            };
+
+            var contractDTO = await ItContractV2Helper.PostContractAsync(token, requestDto);
+
+            var updateRequest1 = new UpdateContractRequestDTO()
+            {
+                Name = CreateName(),
+                ParentContractUuid = parent.Uuid
+            };
+
+            //Act - Update from empty
+            var updatedResponse1 = await ItContractV2Helper.SendPutContractAsync(token, contractDTO.Uuid, updateRequest1);
+
+            //Assert - Update from empty
+            Assert.Equal(HttpStatusCode.OK, updatedResponse1.StatusCode);
+            var updatedContractDTO1 = await ItContractV2Helper.GetItContractAsync(token, contractDTO.Uuid);
+            Assert.Equal(parent.Name, updatedContractDTO1.ParentContract.Name);
```

Comment:
lav en assertcrossreference helper for det er jo generiske sammenligninger du laver